ddtrace/internal/telemetry/writer.py

```diff
@@ -386,6 +386,12 @@ def _telemetry_entry(self, cfg_name: str) -> Tuple[str, str, _ConfigSource]:
         elif cfg_name == "_trace_sampling_rules":
             name = "trace_sampling_rules"
             value = str(item.value())
+        elif cfg_name == "_trace_span_origin_enabled":
+            name = "trace_span_origin_enabled"
+            value = "true" if item.value() else "false"
+        elif cfg_name == "_trace_span_origin_enriched":
+            name = "trace_span_origin_enriched"
```

Comment:
maybe this config should be called the live_debugger? as what it should be in the future is to trigger snapshots based on _dd.p.debug flag